### PR TITLE
Commercial support subpage, footer edits, navbar edits, contact form

### PR DESCRIPTION
**Pull request by klijakub**
Hello, 
we've added:
- a new subpage - /commercial support 
- new subpage - /contact
- misc edits like navbar & footer link, 'new' badge etc
The contact form is linked to the formbucket. We've created a new bucket for test purposes.

### CODE REVIEW

File: docs/_pages/commercial-support/_pricing-table.html
Comment (brikis98) [inline docs/_pages/commercial-support/_pricing-table.html:368]:
```suggestion
				<p><strong>Generic</strong>: the feature must be useful to the majority of Terratest users.</p>
```

File: docs/_pages/contact/_contact-form.html
Comment (brikis98) [inline docs/_pages/contact/_contact-form.html:4]: Where is this URL from? I imagine we'd want a new one in FormBucket specifically for Terratest submissions, right?

File: docs/_pages/contact/index.html
Comment (brikis98) [inline docs/_pages/contact/index.html:26]:
```suggestion
      <a href="mailto:info@terratest.io" >info@gruntwork.io.</a>
```

File: docs/_pages/thanks/index.html
Comment (brikis98) [inline docs/_pages/thanks/index.html:17]:
```suggestion
              >Go to the Terratest home page</a
```

File: docs/assets/js/contact-form.js
Comment (brikis98) [inline docs/assets/js/contact-form.js:8]: Why is this stuff out here and not in the self-calling function below?

File: docs/assets/js/contact-form.js
Comment (brikis98) [inline docs/assets/js/contact-form.js:15]: This doesn't seem to actually submit the form? I see error checking... but where's the submit?

File: docs/_pages/commercial-support/_pricing-table.html
Comment (brikis98) [inline docs/_pages/commercial-support/_pricing-table.html:106]:
```suggestion
					<span>Support via Email</span>
```

File: docs/_pages/commercial-support/_pricing-table.html
Comment (brikis98) [inline docs/_pages/commercial-support/_pricing-table.html:128]:
```suggestion
					<span>Support via real-time chat (Slack)</span>
```

File: docs/_pages/commercial-support/_pricing-table.html
Comment (brikis98) [inline docs/_pages/commercial-support/_pricing-table.html:150]:
```suggestion
					<span>Support via phone/video (Zoom)</span>
```

File: docs/_pages/commercial-support/_pricing-table.html
Comment (brikis98) [inline docs/_pages/commercial-support/_pricing-table.html:216]:
```suggestion
					<span><strong>Prioritized features *</strong></span>
```

File: docs/_pages/commercial-support/_pricing-table.html
Comment (brikis98) [inline docs/_pages/commercial-support/_pricing-table.html:351]:
```suggestion
					<a href="/docs/getting-started/quick-start/" target="_blank" class="btn btn-secondary btn-contact-pricing btn-sm">Start Now</a>
```

File: docs/_pages/commercial-support/_pricing-table.html
Comment (brikis98) [inline docs/_pages/commercial-support/_pricing-table.html:356]: For the "Start Now" and "Contact Sales" buttons, could you:

1. Put more vertical space above them (e.g., `3rem` instead of `1rem`), so they aren't so close to the pricing columns.
1. Add more padding within the buttons. Or, more accurately, make the buttons a bit taller, tiny bit wider, and therefore, make the text in the buttons feel less crammed.

Comment (brikis98) [reply docs/_pages/commercial-support/_pricing-table.html:356]: Also, why `target="_blank"` on these buttons?

File: docs/_pages/thanks/index.html
Comment (brikis98) [inline docs/_pages/thanks/index.html:2]: The thank you page seems to be missing the top nav... Could we put it back?

**Review by brikis98 — COMMENTED** [review]
Thanks for the PR! Looks solid overall. I ran the site and identified a few minor issues, called out in comments.

Also, it looks like the permissions changed on `jekyll-server.sh`, but I think this breaks `docker-compose up` from working. Could you check and revert as necessary?

**Comment by klijakub** [comment]
Thanks for the review @brikis98
We're fixing it right now. 
After we will resolve all issues we will PR Terragrunt also because some of these errors could occur in both projects.

File: docs/_pages/contact/index.html
Comment (klijakub) [reply docs/_pages/contact/index.html:26]: @brikis98 do you want to change mailto attribute also?

File: docs/_pages/contact/_contact-form.html
Comment (klijakub) [reply docs/_pages/contact/_contact-form.html:4]: @brikis98 Yea, we've created a new bucket in your formbucket account

image

File: docs/_pages/contact/index.html
Comment (brikis98) [reply docs/_pages/contact/index.html:26]: Oh, yes, good catch!

File: docs/_pages/contact/_contact-form.html
Comment (brikis98) [reply docs/_pages/contact/_contact-form.html:4]: Oh, that's fantastic! OK, before this merges, can you remember to update that to email `info@gruntwork.io` with form submissions?

**Comment by brikis98** [comment]
> Thanks for the review @brikis98
> We're fixing it right now.
> After we will resolve all issues we will PR Terragrunt also because some of these errors could occur in both projects.

That sounds great, thank you! 👍

File: docs/_pages/contact/_contact-form.html
Comment (klijakub) [reply docs/_pages/contact/_contact-form.html:4]: Sure, we will update. @brikis98
One question - do you want to have a separate bucket for terratest and terragrunt?

Comment (brikis98) [reply docs/_pages/contact/_contact-form.html:4]: Yes, separate buckets for Terragrunt & Terratest is probably the cleanest approach. Thanks!

**Comment by brikis98** [comment]
Is this ready for re-review? Could you mark the comments above resolved when you're done with them? Thx!

**Comment by thedc89** [comment]
> Is this ready for re-review? Could you mark the comments above resolved when you're done with them? Thx!

Hi, this is ready for re-review, all comments should be addressed - not sure how to resolve comments though 🤔
Screenshot 2020-10-21 at 13 18 20

**Comment by brikis98** [comment]
Oh, do you not get a "Resolve conversation" button below each comment?

Screen Shot 2020-10-21 at 12 20 56 PM

**Comment by brikis98** [comment]
Will re-review now!

**Review by brikis98 — COMMENTED** [review]
Ah, crap, it looks like we missed one page: https://terratest.gruntwork.io/docs/community/support/.

Could you update the Commercial Support text to something like:

> Does your company rely on Terratest in production? If so, you can get commercial support directly from Gruntwork, the creators of Terratest! Check out the [Commercial Support Page](#) for more details.

**Comment by ArkuVonSymfon** [comment]
> Ah, crap, it looks like we missed one page: https://terratest.gruntwork.io/docs/community/support/.
> 
> Could you update the Commercial Support text to something like:
> 
> > Does your company rely on Terratest in production? If so, you can get commercial support directly from Gruntwork, the creators of Terratest! Check out the [Commercial Support Page](#) for more details.

Updated

**Comment by brikis98** [comment]
Thanks! Could you merge in the latest from master?

**Comment by brikis98** [comment]
> Thanks! Could you merge in the latest from master?

Bump. I can't merge this in until this is rebased!

**Comment by klijakub** [comment]
> > Thanks! Could you merge in the latest from master?
> 
> Bump. I can't merge this in until this is rebased!

@brikis98 Please try it now, our branch is merged with your master

File: docs/assets/js/contact-form.js
Comment (brikis98) [inline docs/assets/js/contact-form.js:27]:
```suggestion
      submitToFormSpree(data);
```

File: docs/assets/js/contact-form.js
Comment (brikis98) [inline docs/assets/js/contact-form.js:35]: This should prob be read from the HTML form instead of duplicated in the JS code.

**Review by brikis98 — COMMENTED** [review]
Couple more tweaks that I spotted in the Terragrunt PR

File: docs/assets/js/contact-form.js
Comment (klijakub) [reply docs/assets/js/contact-form.js:35]: We're fixing it right now ;)

**Comment by klijakub** [comment]
@brikis98 one more thing about the formbucket and the contact form, we didn't add recaptcha v3 as we did it at gruntwork, but I don't know is it a 'must-have' right now, or maybe you want to check if there will be a lot of spam and then we would add it.

**Comment by brikis98** [comment]
OK, I believe this is good to merge! Could you pull in the latest from master one more time?

**Comment by klijakub** [comment]
> OK, I believe this is good to merge! Could you pull in the latest from master one more time?

Done. Ready to merge:)

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by brikis98** [comment]
This is now announced! https://blog.gruntwork.io/introducing-commercial-support-for-terragrunt-and-terratest-a550f44fde56

Thanks for putting it together 👍

**Pull request merged**
